src/types/events.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+
+type T_EventData = {
```

Comment:
Don't know we we concluded on the naming of the types. Anyway we should follow the same pattern.

I don't like to mark it as a type (prefix with `T_`). Little bit inspired by Martin (https://www.martinfowler.com/bliki/InterfaceImplementationPair.html)